helper/schema/field_reader_config.go

```diff
@@ -136,9 +136,21 @@ func (r *ConfigFieldReader) readMap(k string) (FieldReadResult, error) {
 
 func (r *ConfigFieldReader) readPrimitive(
 	k string, schema *Schema) (FieldReadResult, error) {
+
 	raw, ok := r.Config.Get(k)
+
```

Comment:
Also more idiomatic to not put whitespace here and bump the `if !ok` against the line that sets it.